Toolset/palettes/inspector/editors/com.livecode.pi.gradientramp.behavior.livecodescript

```diff
@@ -87,6 +87,7 @@ on updateColorValue pColor
    lock screen
    lock messages
    set the swatchColor of widget "color" of me to pColor
+   put pColor into field "value" of me
```

Comment:
@montegoulding updateColorDisplay is not called when changing the alpha value by slider but updateColorValue is. So I went with the conservative approach just adding the fix to updateColorValue without fiddling with messaging.
As you say, @livecodeali  will know.